tests/buildtest.cmd

```diff
@@ -11,6 +11,12 @@ set "__PackagesDir=%__ProjectDir%\packages"
 set "__RootBinDir=%__ProjectDir%\bin"
 set "__LogsDir=%__RootBinDir%\Logs"
 
+:: Default to highest Visual Studio version available
+set __VSVersion=vs2015
```

Comment:
The usage message below still says that the default is 2013.  Can you change that?